pkg/kubelet/dockertools/docker.go

```diff
@@ -348,10 +324,10 @@ func milliCPUToShares(milliCPU int64) int64 {
 }
 
 // GetKubeletDockerContainers lists all container or just the running ones.
-// Returns a map of docker containers that we manage, keyed by container ID.
+// Returns a list of docker containers that we manage
 // TODO: Move this function with dockerCache to DockerManager.
-func GetKubeletDockerContainers(client DockerInterface, allContainers bool) (DockerContainers, error) {
-	result := make(DockerContainers)
+func GetKubeletDockerContainers(client DockerInterface, allContainers bool) ([]*docker.APIContainers, error) {
```

Comment:
Yep, I'm just noting for posterity :) It looked funny to me when reviewing
that the call sites didn't change when the type changed.

On Mon, Nov 2, 2015 at 6:16 PM, taotao <notifications@github.com> wrote:

> In pkg/kubelet/dockertools/docker.go
> <https://github.com/kubernetes/kubernetes/pull/16653#discussion_r43698006>
> :
>
> >  // TODO: Move this function with dockerCache to DockerManager.
> > -func GetKubeletDockerContainers(client DockerInterface, allContainers bool) (DockerContainers, error) {
> > -	result := make(DockerContainers)
> > +func GetKubeletDockerContainers(client DockerInterface, allContainers bool) ([]*docker.APIContainers, error) {
>
> Yeah, that's why I want to remove the old type. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16653/files#r43698006>.
>